src/main/java/com/minecolonies/coremod/colony/ColonyView.java

```diff
@@ -64,6 +64,14 @@
      */
     private Set<Block> freeBlocks = new HashSet<>();
 
+    /**
+     * The Set of waypoints.
+     */
+    private Set<BlockPos> wayPoints = new HashSet<>();
+
+    /**
+     * The overall happiness of the colony.
+     */
     private double overallHappiness = 5;
```

Comment:
I don't want to call other classes or create just another constant for this.
I feel like it is okay this way.